test-context/src/lib.rs

```diff
@@ -0,0 +1,151 @@
+use postgresql_embedded::{PostgreSQL, Settings, VersionReq};
+use std::convert::Infallible;
+use std::env;
+use std::env::current_dir;
+use std::io::ErrorKind;
+use std::path::PathBuf;
+use test_context::futures::stream;
+use test_context::AsyncTestContext;
+use tokio::io::AsyncReadExt;
+use tokio_util::bytes::Bytes;
+use tracing::{info_span, instrument, Instrument};
+use trustify_common as common;
+use trustify_module_ingestor::graph::Graph;
+use trustify_module_ingestor::service::{Format, IngestorService};
+use trustify_module_storage::service::fs::FileSystemBackend;
+
+#[allow(dead_code)]
+pub struct TrustifyContext {
+    pub db: common::db::Database,
+    pub graph: Graph,
+    pub storage: FileSystemBackend,
+    pub ingestor: IngestorService,
+    postgresql: Option<PostgreSQL>,
+}
+
+impl TrustifyContext {
+    pub async fn ingest_documents<'a, P: IntoIterator<Item = &'a str>>(
+        &self,
+        paths: P,
+    ) -> Result<(), anyhow::Error> {
+        let workspace_root = find_workspace_root()?;
+        let test_data = workspace_root.join("etc").join("test-data");
+
+        for path in paths {
+            //ingestor.ingest((), None, Format::from_bytes(bytes), bytes)
+            let path = test_data.join(path);
+            let mut file = tokio::fs::File::open(path).await?;
+            let mut bytes = Vec::new();
+            file.read_to_end(&mut bytes).await?;
+
+            let format = Format::from_bytes(&bytes)?;
+
+            let stream = stream::iter([Ok::<_, Infallible>(Bytes::copy_from_slice(&bytes))]);
```

Comment:
I think you can just do this:

```suggestion
            let stream = ReaderStream::new(bytes.as_ref());
```

Reply:
Thanks!